R/R.sublime-syntax

```diff
@@ -3,7 +3,6 @@
 # http://www.sublimetext.com/docs/3/syntax.html
 name: R
 file_extensions:
-  - R
   - r
```

Comment:
I think the capital version should be kept here, so it would be the default when saving a new file: https://cran.r-project.org/doc/manuals/r-release/R-intro.html#Scripting-with-R